news/2018-03-27-the-qat-gazette-march-2018.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "The QAT Gazette: March 2018"
+date: 2018-03-27 09:00:00 +0000
+---
+## Opening 
```

Comment:
Don't typically use an ##Opening header here, since the first few sentences are part of the main page blurb / announcement. If you do want to include it, then this header and the Welcome back...! line need to go right after the banner.

Reply:
done, moved the header after the banner